src/zkevm_specs/evm/execution/signextend.py

```diff
@@ -0,0 +1,41 @@
+from ..instruction import Instruction, Transition
+from zkevm_specs.util import FQ
+
+
+def signextend(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    index = instruction.stack_pop()
+    value = instruction.stack_pop()
+    result = instruction.stack_push()
+
+    is_msb_sum_zero = instruction.is_zero(FQ(sum(index.le_bytes[1:32])))
```

Comment:
I see that `src/zkevm_specs/opcode/signextend.py` had comments explaining the different constraints that are happening.  Could you copy those comments here to keep the same level of documentation?